### ISSUE TITLE
Keep Key issue

### ISSUE THREAD

**Issue by cAP5L0CK** [sparrowwallet/sparrow#219]
Hello,

When trying to create a wallet using the keep key sparrow is failing at the point that it sends the KeepKey the BIP-39 passphrase.

![image](https://user-images.githubusercontent.com/11202204/134815478-3a359476-269f-4484-93d4-b1bbc1feed5d.png)

This happens if a passphrase is sent.

This is the result when trying to set to No Passphrase:
![image](https://user-images.githubusercontent.com/11202204/134815511-3ef1134f-64f3-4804-9f7f-8f6418ace2fd.png)

This is with:
Sparrow 1.5.0
Keep Key Bootloader 1.1.0
Keep Key Firmware 7.2.1
Windows 10

I have tried resetting the hardware device completely.

Interestingly, this is a nearly identical issue to one happening with Specter Wallet as well.
https://github.com/cryptoadvance/specter-desktop/issues/1219

This Keepkey does work fine with Electrum 4.1.5 including wallets with a passphrase and without, otherwise I might think it was the hardware itself. 

Let me know how I can help.

**Comment by craigraw** [comment]
Since both Specter and Sparrow use HWI, so it's probably an HWI issue. 

Unfortunately Keepkey has been off the market for some time (and I don't have one) so it's difficult to test. But I would file an issue at https://github.com/bitcoin-core/HWI/issues - that's where this would need to be addressed.

**Comment by cAP5L0CK** [comment]
Thanks craigraw.  I have created a issue under that project.  It's a shame in some ways, as the KeepKey is a bit of a sleeper.  I am assuming it is a Trezor fork, and it works quite well.  I am a HW wallet junkie. ;)  Speaking of which will you consider adding Jade?

**Comment by craigraw** [comment]
> Speaking of which will you consider adding Jade?

Absolutely. Waiting on https://github.com/bitcoin-core/HWI/pull/475

**Comment by 6102bitcoin** [comment]
Marked as wontfix as it's likely a HWI bug

**Comment by craigraw** [comment]
Closing this off - my advice to anyone using Keepkey is to move to another hardware wallet, as I don't believe HWI intend to maintain support for it.
